LanguageFeatures/Patterns/variable_A01_t01.dart

```diff
@@ -49,11 +49,11 @@ main() {
   Expect.equals(5, c1);
   Expect.equals("6", d1);
 
-  var [int a2, b2] = [1, 3.14];
+  var [int a2, b2] = [1, 2 as num];
   Expect.equals(1, a2);
-  Expect.equals(3.14, b2);
+  Expect.equals(2, b2);
   a2.expectStaticType<Exactly<int>>();
-  b2.expectStaticType<Exactly<double>>();
+  b2.expectStaticType<Exactly<int>>();
```

Comment:
Have to ask: Why isn't `[1, 2 as num]` inferred as a `<num>[...]` list literal and subsequently `[int a2, b2]` inferred as a `<num>[...]` list pattern, and `int a2` is then a compile-time error because it's a variable pattern with a built-in downcast in a declaration context?

Reply:
Indeed. Thank you! Fixed. Please rereview

Reply:
Thx!